FreeRTOS/FreeRTOS
# CBMC proof for ulARPRemoveCacheEntryByMac

Description
-----------
This PR adds a CBMC proof for the `ulARPRemoveCacheEntryByMAC` function present in FreeRTOS_ARP.c.

Test Steps
-----------
- Download the repo with `--recurse-submodules`. Switch to the correct branch.
- Switch to `FreeRTOS-Plus/Test/CBMC/proofs` directory and run `python3 prepare.py`.
- Move to `ARP/ulARPRemoveCacheEntryByMAC` directory. Run `make cbmc`.
- Open the cbmc.txt file generated by the above command.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
